repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/homepage-main.ftl

```diff
@@ -14,87 +14,84 @@
 <#else>
```

Comment:
@sankar-shunmuga - do you think it will be Ok to remove the support for non-image hero banner as `Hero image` is a mandatory field for the Home page? Also, `Hero image` seems to exist on all channel Home pages.

Reply:
Resolved